From: KevinDuan <duankaiwen@kylinos.cn>
Date: Mon, 17 Aug 2026 08:08:40 +0000
Subject: Translated using Weblate (Japanese)

Currently translated at 100.0% (12 of 12 strings)

Translation: openkylin-huanghe/wlanconnect
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/wlanconnect/ja/
---
 plugins/wlanconnect/translations/ja.ts | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/plugins/wlanconnect/translations/ja.ts b/plugins/wlanconnect/translations/ja.ts
index fcfbcc1..b54c13b 100644
--- a/plugins/wlanconnect/translations/ja.ts
+++ b/plugins/wlanconnect/translations/ja.ts
@@ -6,12 +6,12 @@
     <message>
         <location filename="../../component/AddBtn/addnetbtn.cpp" line="44"/>
         <source>Add Others</source>
-        <translation type="unfinished"></translation>
+        <translation>他のネットワークに参加する</translation>
     </message>
     <message>
         <location filename="../../component/AddBtn/addnetbtn.cpp" line="48"/>
         <source>Add WiredNetork</source>
-        <translation type="unfinished"></translation>
+        <translation>有線ネットワークを追加</translation>
     </message>
     <message>
         <source>Add WiredNetwork</source>
@@ -23,7 +23,7 @@
     <message>
         <location filename="../wlanconnect.ui" line="14"/>
         <source>WlanConnect</source>
-        <translation type="unfinished"></translation>
+        <translation>無線ローカルエリアネットワーク</translation>
     </message>
     <message>
         <location filename="../wlanconnect.ui" line="35"/>
@@ -31,19 +31,19 @@
         <location filename="../wlanconnect.cpp" line="283"/>
         <location filename="../wlanconnect.cpp" line="285"/>
         <source>WLAN</source>
-        <translation type="unfinished"></translation>
+        <translation>無線ローカルエリアネットワーク</translation>
         <extra-contents_path>/wlanconnect/WLAN</extra-contents_path>
     </message>
     <message>
         <location filename="../wlanconnect.ui" line="94"/>
         <source>open</source>
-        <translation type="unfinished"></translation>
+        <translation>発足</translation>
     </message>
     <message>
         <location filename="../wlanconnect.ui" line="147"/>
         <location filename="../wlanconnect.cpp" line="280"/>
         <source>Advanced settings</source>
-        <translation type="unfinished"></translation>
+        <translation>詳細設定</translation>
         <extra-contents_path>/wlanconnect/Advanced settings&quot;</extra-contents_path>
     </message>
     <message>
@@ -57,36 +57,36 @@
     <message>
         <location filename="../wlanconnect.cpp" line="137"/>
         <source>Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>セットアップ</translation>
     </message>
     <message>
         <location filename="../wlanconnect.cpp" line="140"/>
         <source>Settings desktop message</source>
-        <translation type="unfinished"></translation>
+        <translation>デスクトップ通知の設定</translation>
     </message>
     <message>
         <location filename="../wlanconnect.cpp" line="278"/>
         <source>Add Others</source>
-        <translation type="unfinished"></translation>
+        <translation>他のネットワークに参加する</translation>
         <extra-contents_path>/wlanconnect/Add Others&quot;</extra-contents_path>
     </message>
     <message>
         <location filename="../wlanconnect.cpp" line="301"/>
         <location filename="../wlanconnect.cpp" line="1299"/>
         <source>No wireless network card detected</source>
-        <translation type="unfinished"></translation>
+        <translation>ワイヤレスネットワークカードは検出されません</translation>
     </message>
     <message>
         <location filename="../wlanconnect.cpp" line="433"/>
         <location filename="../wlanconnect.cpp" line="1137"/>
         <location filename="../wlanconnect.cpp" line="1206"/>
         <source>connected</source>
-        <translation type="unfinished"></translation>
+        <translation>接続</translation>
     </message>
     <message>
         <location filename="../wlanconnect.cpp" line="1075"/>
         <source>card</source>
-        <translation type="unfinished"></translation>
+        <translation>ネットワークカード</translation>
     </message>
 </context>
 </TS>
